.circleci/config.yml

```diff
@@ -21,8 +21,7 @@ jobs:
       - run:
           name: Run lint
           command: |
-            source .venv/bin/activate
-            make docs-lint
+            poetry run make docs-lint
```

Comment:
This seems a little silly to me, can we use either poetry as the command runner or make? Needing to invoke both like this seems excessive.

Reply:
The reason to use `poetry run` here is that it will run the target in the virtual environment, which you typically would `poetry shell` into first when running the `make` target locally. I would prefer not to copy the specific `sphinx-build` invocations in two places (DRY).

We could skip the creation of a virtualenv (`poetry config virtualenvs.create false`), but that gets hackish in its own ways -- `poetry` will then install the dependencies in `site-packages`, which is not added to `sys.path` in Debian and would require additional `PYTHONPATH` munging.

Reply:
I think the easiest solution would be to have the Makefile call `poetry run sphinx-build...` and then `make docs-lint`, etc. will all continue to work whether or not you activate the venv via `poetry shell` (it would be nice if we didn't have to activate the venv before doing things and it all happened transparently).

(Unfortunately `poetry run` can't execute arbitrary commands yet: https://github.com/python-poetry/poetry/issues/2496)

Reply:
That sounds reasonable. This is done now. It also allowed us to remove one step from the `README`.